src/main/java/com/flightstats/hub/cluster/DistributedLeaderLockManager.java

```diff
@@ -41,10 +41,10 @@ public boolean tryAcquireLock(LeadershipLock leadershipLock, long waitTimeout, T
                 log.debug("acquired {} {}", leadershipLock.getLockPath(), leadership.hasLeadership());
                 return true;
             } else {
-                log.debug("unable to acquire {} ", leadershipLock.getLockPath());
+                log.warn("unable to acquire {} ", leadershipLock.getLockPath());
             }
         } catch (Exception e) {
-            log.warn("oh no! issue with " + leadershipLock.getLockPath(), e);
+            log.error("oh no! issue with " + leadershipLock.getLockPath(), e);
             release(leadershipLock);
```

Comment:
you can parameterize these log statements instead of string concatenation. Look at the example on line 44. This applies to all other log statements.